packages/core/scripts/identifier-umip/1_Propose.js

```diff
@@ -45,7 +57,8 @@ async function runExport() {
     };
   });
 
-  await governor.propose(transactions, { from: proposerWallet });
+  await gasEstimator.update();
+  await governor.propose(transactions, { from: proposerWallet, gasPrice: gasEstimator.getCurrentFastPrice() });
```

Comment:
Unrelated, but should we have consistency in how transaction(s) are passed to the proposal method? We have three slightly different ways of doing this: `transactions` (identifier), `transactionList` (collateral), and a hardcoded array with a single element (creator).

Reply:
Not super opinionated. The creator one only allows a single transaction, so it is a little different from the others.

I think there's more stuff we could factor out and synchronize across these scripts, so I might suggest doing a refactor of all these scripts in a follow up.